.dockerignore

```diff
@@ -0,0 +1,7 @@
+*
```

Comment:
I'd like to keep the `.git` folder. Go automatically extracts VCS information from it and adds those to the resulting binary. See https://github.com/restic/restic/blob/481fcb9ca78d009bea9b19aed6ac22f323838c57/.github/workflows/docker.yml#L59